infra/hooks/predown.ps1

```diff
@@ -0,0 +1,8 @@
+# Load the azd environment variables
+./infra/hooks/load_azd_env.ps1
```

Comment:
You will need to use the $DIR logic (like below) when calling the files because of the way hooks works to make it compatible across platforms.

Please double check this across all hook scripts.